src/logic/operate.js

```diff
@@ -0,0 +1,36 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let result = Big(0);
+  const a = Big(numberOne);
+  const b = Big(numberTwo);
+
+  switch (operation) {
+    case '+':
+      result = a.plus(b);
+      break;
+
+    case '-':
+      result = a.minus(b);
+      break;
+
+    case 'X':
+      result = a.times(b);
+      break;
+
+    case '/':
+      result = a.div(b);
+      break;
+
+    case '%':
+      result = a.div(100);
+      break;
+
+    default:
+      result = a;
+      break;
```

Comment:
It will be great to throw an error as the default return value. This is because by this point a user either added an operation that is not supported in your calculator. So instead of just returning a value consider throwing an error. For example: ``` throw Error(`Unknown operation '${operation}'`); ```